### ISSUE TITLE
Content association Test

### ISSUE THREAD

**Issue by vsiless** [FAI3network/ICP_MVP#35]

**Comment by hhaamm** [comment]
After reading the paper carefully, they use logits, not text for this test. So I'm moving to https://github.com/FAI3network/ICP_MVP/issues/36.
